src/development/dev-tests.md

```diff
@@ -15,7 +15,7 @@ review:
 Developers are responsible for testing the code they write. We require two types of test as a minimum:
 
 * __Unit test__
-  A self contained test for a unit of functionality, such as a Java class or a Javascript function.
+  A self contained test for a unit of functionality, such as a Java class or a Javascript function, or a behaviour spanning many such files.
```

Comment:
I'm not sure I agree a behavioural test spanning multiple files is a `unit` test.
Shall we discuss in the dev community?